src/components/AppLoader.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import { SyncIndicator } from '@1hive/1hive-ui'
+
+export default function AppLoader() {
+  return <SyncIndicator visible label="Loading app" />
```

Comment:
do you think we need this file just for this?

Reply:
Yes, this is just a placeholder for now. But Fiore is working on a custom loader so i though it was a good idea to already separate the component

Reply:
cooool